wiki/Skinning/Interface/en.md

```diff
@@ -464,947 +610,1593 @@ _See also: [Offset Wizard](/wiki/Offset_Wizard)._
+
+| Versions | Animatable | Beatmap Skinnable | Blend Mode | Origin | Suggested SD Size |
+|:-:|:-:|:-:|:-:|:-:|:-:|
+| All | ![Yes][true] (see notes) | ![Yes][true] | Normal | Centre | - |
+
+Notes:
+
+- Animation name: `hit300k-{n}.png`
+- Animation rate is fixed to 60 FPS.
+- If animation is used:
+  - animation does not loop, but the last frame persists until it fades out.
+  - single frame behaviour is not used.
+- This is not shown on the ranking screen.
 
 ### Input Overlay
 
-![](img/inputoverlay-background.png "inputoverlay-background.png")
+`inputoverlay-background.png`
 
-| Animatable   | Beatmap Skinnable |
-|:------------:|:-----------------:|
-| ![No][false] | ![Yes][true]      |
+![](img/inputoverlay-background.png)
 
-**Notes:**
+| Versions | Animatable | Beatmap Skinnable | Blend Mode | Origin | Suggested SD Size |
+|:-:|:-:|:-:|:-:|:-:|:-:|
+| All | ![No][false] | ![Yes][true] | Normal | TopRight | 193x55 |
 
-- This is used in [osu!standard](/wiki/osu!standard) and [osu!catch](/wiki/osu!catch).
-- Toggleable in the [options](/wiki/options).
-- This is rotated 90 degrees clockwise.
-- Stretched by 1.05x ingame.
+Notes:
+
+- This element is positioned at 320px height.
+- Since the image is rotated, the origin on the image itself is TopLeft.
+- This element is used in [osu!standard](/wiki/osu!standard) and [osu!catch](/wiki/osu!catch).
+- This element is rotated 90 degrees clockwise and stretched by 1.05x in-game.
+- Must be enabled in the [options](/wiki/options) to see.
 
 ---
 
-![](img/inputoverlay-key.png "inputoverlay-key.png")
+`inputoverlay-key.png`
 
-| Animatable   | Beatmap Skinnable |
-|:------------:|:-----------------:|
-| ![No][false] | ![Yes][true]      |
+![](img/inputoverlay-key.png)
 
-**Notes:**
+| Versions | Animatable | Beatmap Skinnable | Blend Mode | Origin | Suggested SD Size |
+|:-:|:-:|:-:|:-:|:-:|:-:|
+| All | ![No][false] | ![Yes][true] | Multiplicative | Centre | 43x46 |
+
+Notes:
 
 - This is used in [osu!standard](/wiki/osu!standard) and [osu!catch](/wiki/osu!catch).
+- Positioning varies for each key:
+  - 24px away from screenborder
+  - K1/L: at 350px height
+  - K2/R: at 398px height
+  - M1/D: at 446px height
+  - M2: at 492px height
 - Toggleable in the [options](/wiki/options).
-- Expands whenever the keys is pressed.
+- Shrinks briefly when the keys are pressed.
+- Tinting varies by button location and state:
+  - White, if key is not pressed.
+  - Yellow, if the key is pressed and locationed on the top half.
+  - Purple, if the key is pressed and locationed on the bottom half.
 
 ### Pause Screen
 
-_`pause-overlay.png`_ (transparent; not included)
+`pause-overlay.png`
 
-| Animatable   | Beatmap Skinnable |
-|:------------:|:-----------------:|
-| ![No][false] | ![Yes][true]      |
+| Versions | Animatable | Beatmap Skinnable | Blend Mode | Origin | Suggested SD Size |
+|:-:|:-:|:-:|:-:|:-:|:-:|
+| All | ![No][false] | ![Yes][true] | Normal | Centre | 1366x768 |
 
-**Notes:**
+Notes:
```

Comment:
`pause-overlay` and `fail-background` can also be `.jpg-files`

Reply:
additional notes: preferences `.png` over `.jpg`